metering/configuring_metering/metering-common-config-options.adoc

```diff
@@ -82,22 +82,22 @@ spec:
   reporting-operator:
     spec:
       nodeSelector:
-        "node-role.kubernetes.io/infra": "true"
+        "node-role.kubernetes.io/infra": ""
```

Comment:
Let's add an annotation to the end of these lines and call out how the label works below the code block.

"node-role.kubernetes.io/infra": "" **<1>**